README.md

```diff
@@ -79,22 +73,22 @@ Apache Airflow is tested with:
 
 Visit the official Airflow website documentation (latest **stable** release) for help with [installing Airflow](https://airflow.apache.org/installation.html), [getting started](https://airflow.apache.org/start.html), or walking through a more complete [tutorial](https://airflow.apache.org/tutorial.html).
 
-> Note: If you're looking for documentation for master branch (latest development branch): you can find it on [ReadTheDocs](https://airflow.readthedocs.io/en/latest/).
+> Note: If you're looking for documentation for the master branch (latest development branch): you can find it on [ReadTheDocs](https://airflow.readthedocs.io/en/latest/).
 
 For more information on Airflow's Roadmap or Airflow Improvement Proposals (AIPs), visit the [Airflow Wiki](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Home).
 
 Official Docker (container) images for Apache Airflow are described in [IMAGES.rst](IMAGES.rst).
 
 ## Installing from PyPI
 
-We publish Apache Airflow as `apache-airflow` package in PyPI. Installing it however might be sometimes tricky
```

Comment:
`an` isn't the correct article in this case, since `apache-airflow` is the name of the specific package, and does not refer to a general type or group of packages.